misc/python/materialize/feature_benchmark/scenario.py

```diff
@@ -47,12 +46,12 @@ def n(self) -> int:
         return self._n
 
     @staticmethod
-    def name_with_scale(cls: type["Scenario"], num: int, params_dict: dict) -> str:
+    def name_with_scale(class_: type["Scenario"], num: int, params_dict: dict) -> str:
```

Comment:
Interesting fix 😀

Reply:
Yeah, it seems a bit of a picky lint, but I can also see its value for readability. Conventionally in Python `cls` refers to an instance of the defining class, and is meant to be used with class methods; this could be confusing if it were left as-is.